Makefile

```diff
@@ -85,6 +94,7 @@ test-e2e:
 	LOCAL_IMAGE=$(S2I_TEST_IMAGE_PYSPARK) make -f Makefile.pyspark build
 	LOCAL_IMAGE=$(S2I_TEST_IMAGE_JAVA) make -f Makefile.java build
 	LOCAL_IMAGE=$(S2I_TEST_IMAGE_SCALA) make -f Makefile.scala build
+	LOCAL_IMAGE=$(S2I_TEST_IMAGE_SPARKLYR) make -f Makefile.sparklyr build
 	test/e2e/run.sh
```

Comment:
hmm, just an aside, I bet this is supposed to be "test/e2e/run.sh operations".  Not part of this PR but I'll look.